app/assets/javascripts/discourse/app/templates/invites/show.hbs

```diff
@@ -74,7 +74,9 @@
               <div class="input name-input">
                 <label for="new-account-name">
                   {{i18n "invites.name_label"}}
-                  <span class="required">*</span>
+                  {{#if this.siteSettings.full_name_required}}
```

Comment:
This is a bit odd, shouldn't it be `{{#if siteSettings. full_name_required}}` to match other templates? I am surprised `this` works.

Reply:
@eviltrout Yes, it's best to match other templates. Have modified this. I believe `this.siteSettings` works because it's available through the controller. Please correct me if I am wrong.